cms/templates/cms/program_page.html

```diff
@@ -11,7 +12,8 @@
   <div class="app-media layout-boxed">
     <div class="main-content-wrapper">
       <section class="main-content">
-        <div class="banner-wrapper banner-sub banner-program-1">
+      {% image page.background_image fill-1310x613 as tmp_photo %}
```

Comment:
1. The `image` tag  takes two arguments `{% image [image] [resize-rule] %}`, otherwise would not work.
2. I will rename the variable to `background_image`.